src/Markdown.MAML/Parser/MarkdownParser.cs

```diff
@@ -148,7 +148,15 @@ private void InitializePatternList()
 
                         {   "italic",
                             @"\*(.+?)\*",
-                            this.CreateItalicSpan }
+                            this.CreateItalicSpan },
+
+                        {   "bold2",
+                            @"(?<![a-zA-Z0-9])__(.+?)__(?![a-zA-Z0-9])",
```

Comment:
This should be exectly the same as original bold regex:
```
@"_(.+?)_"
```
why this difference?
It also allows you to reuse the same `this. CreateBoldSpan`, without creating a new function